webapp/src/app/admin/layout.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="container space-y-4">
```

Comment:
This component/layout is not responsive for mobile screens, I guess you did not test it. Try using storybook more to test the components.